keyboards/handwired/aplx2/README.md

```diff
@@ -0,0 +1,34 @@
+# Aplx2
+
+![Front](https://i.imgur.com/cN4WfaQ.jpg)  
+   
+A simple 2% keyboard for osu!. Designed to be assembled as a sandwich with a Pro Micro, using M3 x 16mm screws, your choice of MX- or Alps-compatible switches, and 1N4148 diodes.   
+
+**Layout:**  
+The default layout is: Z , X .
+Used pins can be found in `config.h`. All ProMicro pins can be found [here](https://golem.hu/article/pro-micro-pinout/).
```

Comment:
Personally, I'd prefer this link, as it's from SparkFun:
```suggestion
Used pins can be found in `config.h`. All ProMicro pins can be found [here](https://cdn.sparkfun.com/datasheets/Dev/Arduino/Boards/ProMicro16MHzv1.pdf).
```

Reply:
sure why not!